CMakeLists.txt

```diff
@@ -229,13 +229,18 @@ if(BUILD_JAVA)
   add_subdirectory(java)
 endif()
 
-# Check for FLTK
-set(FLTK_SKIP_FLUID TRUE)
-set(FLTK_SKIP_OPENGL TRUE)
-set(FLTK_SKIP_FORMS TRUE)
-find_package(FLTK)
+option(BUILD_VIEWER "Build TigerVNC viewer" ON)
+if(BUILD_VIEWER)
+  # Check for FLTK
+  set(FLTK_SKIP_FLUID TRUE)
+  set(FLTK_SKIP_OPENGL TRUE)
+  set(FLTK_SKIP_FORMS TRUE)
+  find_package(FLTK REQUIRED)
+else()
+  set(FLTK_FOUND FALSE)
```

Comment:
Can we avoid this? It was barely used before, and it is a bit odd compared to everything else we have.

We should be able to avoid it by using `BUILD_VIEWER` and `if(DEFINED FLTK_LIBRARIES)` instead.